Mail/Views/Alerts/CreateFolderView.swift

```diff
@@ -125,6 +157,14 @@ struct CreateFolderView: View {
             return
         }
 
+        if folder != nil {
+            if folderName == folder!.name {
+                isInitialName = true
+            } else {
+                isInitialName = false
+            }
+        }
```

Comment:
I don't remember why. But is there a reason for this ?
We can simply use the same `nameAlreadyExists` error. It will be checked it the next lines.